src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx

```diff
@@ -12,6 +12,76 @@ import {Client} from 'app/api';
 import {update} from 'app/actionCreators/projects';
 import overflowEllipsis from 'app/styles/overflowEllipsis';
 
+class PlatformList extends React.Component {
+  static propTypes = {
+    platforms: PropTypes.arrayOf(PropTypes.string),
+  };
+
+  getIcon(platform) {
+    return (
+      <StyledPlatformIconWrapper key={platform} className={platform}>
+        <StyledPlatformIcon className={`platformicon platformicon-${platform}`} />
+      </StyledPlatformIconWrapper>
+    );
+  }
+
+  getIcons(platforms) {
+    return (
+      <Flex direction="row-reverse" m={18}>
```

Comment:
is `18` on purpose here? what about `2`?

Reply:
Good catch, I think the margins are a couple of pixels off